src/sbuf/SBuf.h

```diff
@@ -192,6 +193,9 @@ class SBuf
     /// Append a single character. The character may be NUL (\0).
     SBuf& append(const char c);
 
+    /// Append a single character, compatibly with std::string::push_back()
+    void push_back(const char c) { append(c); }
```

Comment:
Please do these the other way around. Implementing our custom `append()` API in terms of the STL one.

```suggestion
    /// \copydoc push_back(const char c)
    /// The character may be NUL (\0).
    SBuf& append(const char c) { push_back(c); return *this; }

    /// Append a single character, compatibly with std::string::push_back()
    void push_back(const char c);
```

This will need relevant SBuf.cc changes.

Reply:
done